content/topics/How-to-ask-for-help-with-your-code/_index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: how to ask for help with your code
+ready: true
+---
+
+A big part of learning to be a programmer of any kind is knowing how to get help. This means knowing how to ask questions.
+Your questions should be short and clear. Your question should include all the information that is needed to answer the question.
+Practice asking good questions. It's like a superpower. If you are easy to help then people will help you.<br/><br/>
+
+### These also include:
+
+#### how to ask good questions on slack:
+
+* all Slack questions should be addressed on the slack **Tech Assist** channel.
+ - This helps you get to a broad audience from the tech department (.e.g Recruits, Managers and Alumni).
+ - Please don't send screenshots what you can do is use backtick (` `) characters to represent a code form.
```

Comment:
Please revise sentence structure. Punctuation is your friend. 

Also, there is inconsistent use of bullet points here. * and -. And different indentation. I wonder what it will look like when rendered. Better to just make it consistent